## 25_entropy.patch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove code that called netstat to gain so called entropy. It's pretty
## DP: useless on a Linux system. Thanks Wichert Akkerman. bz#51429.

diff --git a/mozilla/security/nss/lib/freebl/unix_rand.c b/mozilla/security/nss/lib/freebl/unix_rand.c
index c8344b9..6c019da 100644
--- a/mozilla/security/nss/lib/freebl/unix_rand.c
+++ b/mozilla/security/nss/lib/freebl/unix_rand.c
@@ -849,7 +849,11 @@ safe_pclose(FILE *fp)
 /* Fork netstat to collect its output by default. Do not unset this unless
  * another source of entropy is available
  */
+#ifndef LINUX
 #define DO_NETSTAT 1
+#else
+#undef DO_NETSTAT 
+#endif
 
 void RNG_SystemInfoForRNG(void)
 {
